sections/introduction.tex

```diff
@@ -14,6 +14,7 @@
 \vfill
 \columnbreak
 \note{5}{Exceptions and notes with \hyperlink{Heroes of Might and Magic III}{amber colored hyperlinks} are explained in boxes like this one.}
+\note{6}{Conflicting rules changes on components follow this priority: Player Cards, Unit Cards, Town Boards, Mission Book, this rule book.}
```

Comment:
Please combine those boxes. Having separate ones so close to each other doesn't look good.